doc/releases/changelog-dev.md

```diff
@@ -216,38 +216,47 @@
   operation on our quantum circuits for both qubit and CV devices.
   [(#1829)](https://github.com/PennyLaneAI/pennylane/pull/1829)
 
-* For Hamiltonians whose eigenvalue frequency spectrum is known, `qml.gradients.get_shift_rule` is
-  a function that computes the generalized parameter shift rules for the time evolution.
+* Given an operator of the form :math:`U=e^{iHt}` where :math:`H` has commuting terms and known
+  eigenvalues (and thus known frequency spectrum),
+  `qml.gradients.generate_shift_rule` computes the generalized parameter shift rules for determining
+  the gradient of the expectation value :math:`\langle 0|U(t)^\dagger \hat{O} U(t)|0\rangle` on
+  hardware.
   [(#1788)](https://github.com/PennyLaneAI/pennylane/pull/1788)
+  [(#1932)](https://github.com/PennyLaneAI/pennylane/pull/1932)
 
-  Given a Hamiltonian's frequency spectrum of `R` unique frequencies, `qml.gradients.get_shift_rule`
-  returns the parameter shift rules to compute expectation value gradients of the Hamiltonian's
+  Assuming that :math:`H` has `R` unique frequencies, `qml.gradients.generate_shift_rule`
```

Comment:
The frequencies of an operator are not a standard term. I fear we have to link to something where people can find an explanation?

Sorry for not picking this up before

Reply:
Good point, I suppose we use it as a shortcut for

> Fourier-series frequencies of the function `f(t) = <0|U(t)† O U(t)|0>`, for some observable `O` where `U(t)=exp(iHt)`.

Reply:
@mariaschuld I've updated the changelog here: 680164c Let me know if that reads better!